indicator_templates/Enumerations/PopulationEnumeratedByArea.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace App\IndicatorTemplates\Enumerations;
+
+use App\Http\Livewire\Chart;
+use App\Services\Interfaces\BarChart;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+/**
+ * Population enumerated by area
+ * A BarChart showing the population enumerated by area, with a line chart showing the target population to be enumerated.
+ *
+ * @category BarChart
+ */
+abstract class PopulationEnumeratedByArea extends Chart implements BarChart
+{
+    public const POPULATION_DATA_KEY =[ 'area_code', 'population'];
+    /**
+     * @return \Illuminate\Support\Collection
+     * $data = [
+     *     [
+     *          'area_name' => 'EAS 1',
+     *          'area_code' => 'EAS 1',
+     *
+     *          'population' => 100,
+     *    ],
+     *   [
+     *
+     *         'area_name' => 'EAS 2',
+     *        'population' => 200,
+     *   ],
+     * ];
+     */
+    abstract protected function getPopulationCountedData(array $filter = []);
```

Comment:
While other already implemented methods could be called anything, I suggest we come up with a standard name for abstract methods so that the dev can gest used to the methods that need to be implemented. 
It's not great that they have to grok new names for implementing each template